### PR TITLE
Dockerfile update

### PR DESCRIPTION
**Pull request by anoopwebs**
**What this PR does / why we need it**:
People outside caicloud can't build docker image locally due to dependency with private base image. This PR to solve that issue. Also, this change helps to reduce image size.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: store.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func NewEventStore(client *k8s_client.Client, init, max time.Duration) (*EventSt
 		events: prometheus.NewDesc(
 			"kubernetes_events",
 			"State of kubernetes events",
-			[]string{"event_namespace", "event_name", "event_kind", "event_reason", "event_type", "event_subobject", "event_source"},
+			[]string{"event_namespace", "event_name", "event_kind", "event_reason", "event_type", "event_subobject", "event_message", "event_source"},
```
Comment (supereagle) [inline store.go]: Is this already included in #9?

Comment (anoopwebs) [reply store.go]: yes, it is. my fork didn't have those

**Comment by emillg** [comment]
@supereagle @anoopwebs When can this new image be released? Can it have a release label?

**Comment by supereagle** [comment]
@anoopwebs  I think you need to rebase this PR, then it will not included the code of #9.

**Comment by supereagle** [comment]
@emillg  Will release one version after this PR merged.

**Comment by anoopwebs** [comment]
> @anoopwebs I think you need to rebase this PR, then it will not included the code of #9.

@supereagle  done!

File: store.go [hunk 2]
```diff
@@ -160,6 +160,7 @@ func (es *EventStore) Scrap(ch chan<- prometheus.Metric) error {
 			event.Reason,
 			event.Type,
 			event.InvolvedObject.FieldPath,
+			event.Message,
```
Comment (emillg) [inline store.go]: @anoopwebs When can this change be released?

**Comment by supereagle** [comment]
@emillg Have create release [v0.1.0](https://github.com/caicloud/event_exporter/releases/tag/v0.1.0)

**Comment by emillg** [comment]
Thanks @supereagle

**Comment by emillg** [comment]
hi @supereagle, it looks like the Docker image doesn't have that release version?

```
$ docker pull cargo.caicloud.io/sysinfra/event-exporter:v0.1.0
Error response from daemon: manifest for cargo.caicloud.io/sysinfra/event-exporter:v0.1.0 not found
```

**Pull request merged**